munit/shared/src/main/scala/munit/FileLayout.scala

```diff
@@ -0,0 +1,75 @@
+package munit
+
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.StandardOpenOption
+import java.nio.file.Path
+
+trait FileLayout {
```

Comment:
What do you think about turning this into an object?
```scala
object FileLayouts {
  def parse(..)
  def write(..)
}
```
This could help improve the completion experience in IDEs and also avoids adding `{parse,write}FileLayout` to the `FunSuite` scope 🤔

Reply:
I initially had it implemented like this, but then changed my mind. I thought it would be nice to avoid an import to use `FileLayout` methods so I decided to make it a trait and mix it in, to make it as usable as `Assertions`

We could also do

```scala
trait FileLayout {
  object FileLayout {
    def parse(...)
    def write(...)
  }
}
```

or something like that, what do you think?

Reply:
I'm not convinced this utility is used enough to merit inclusion in the `FunSuite` scope, unlike assertions. I think it's also desirable to have access to this object outside of `FunSuite`. We can have a `val FileLayout = munit.FileLayout` forwarder inside `FunSuite` but I think we're entering a slippery slope then and end up adding every new feature into the `FunSuite` scope